modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -353,7 +353,7 @@ package object engine {
 
   def pure[A](a: A): HandleP[A] = Applicative[HandleP].pure(a)
 
-  private val unit: HandleP[Unit] = pure(Unit)
+  private val unit: HandleP[Unit] = pure(())
```

Comment:
Right, `Unit` is the type. Apparently this causes confusions on haskellers where both are the same